lib/link_thumbnailer/scrapers/default/body.rb

```diff
@@ -0,0 +1,27 @@
+# frozen_string_literal: true
+
+require 'link_thumbnailer/scrapers/default/base'
+
+module LinkThumbnailer
+  module Scrapers
+    module Default
+      class Body < ::LinkThumbnailer::Scrapers::Default::Base
+
+        def value
+          model.to_s
+        end
+
+        private
+
+        def model
+          modelize(node)
+        end
+
+        def node
+          document.css('body').css('p')
```

Comment:
Have you considered also parsing h1-h6 tags, plus all others that might be used nowadays (unsure which, but maybe cite, blockquote, q,…)?